server/common/config.go

```diff
@@ -6,6 +6,8 @@ import (
 
 const (
 	TimeFormat = time.RFC3339
+	Cooldown   = time.Duration(5 * time.Minute)
```

Comment:
I think this needs to be 3 seconds now

Reply:
sure we can play around w/ that.  ideally we have a "debug" mode and a "release" mode so we don't have to make code changes to swap between short and long cooldown.

Reply:
i'll update this to use 3 seconds, let's create an issue for this though.